docs/onboarding_best_practices.rst

```diff
@@ -0,0 +1,165 @@
+
+5. Sorted column
+###################
+Sort data on column if it will always be present in the queries. **Sorted index performs superior than inverted index.** 
+
+- For offline, Pinot cannot sort the data in segments that are pushed to pinot. This needs to be done prior to the segment creation, in the data preparation job. If the data is sorted before segment creation, the segment creation will detect and maintain this sorted column. 
+- For realtime, set the column in “sortedColumn” in table config, and it will be sorted in Pinot when data is converted to a Pinot segment.
+
+6. Multi value columns
+#######################
+Pinot supports multi value columns. Arrays in an avro will be treated as multi value columns, by setting `“singleValueField” : “false”` in the dimensionFieldSpec in Pinot schema. However, in general they are more costly for query execution, and ensure that they are being used only when needed.
+
+7. Inverted indexes
+###################
+For columns that will be used fairly frequently in your queries, enable inverted indexes. Inverted indexes are **more efficient and effective for high cardinality columns.** 
+
+8. Partitioning
+###################
+- For offline, if the data has been partitioned, Pinot can leverage this information for querying. This can be done by setting the partitioning function, number of partitions and column name in the segmentPartitionConfig in the table config 
+- If the data is partitioned when splitting into the stream partitions, Pinot can leverage this information when querying, and prune segments accordingly, effectively speeding up queries. This can be done by setting the partitioning function, number of partitions and column name in the segmentPartitionConfig.
+
+9. Number of segments
+######################
+Avoid creating too many small Pinot segments. Pinot’s query execution happens at the segment level, which can suffer from a substantial query planning overhead if faced with too many small segments.This can be controlled by the number of input files fed to the segment creation job. In realtime, this can be controlled by tuning the rows, time or size thresholds as described in :ref:`pluggable-streams`
+
+
+
+
+Realtime specific considerations
+*********************************
+
+1. Time column
+###################
+Time column is a must for hybrid table
```

Comment:
time column must be defined in both offline and realtime table configs to be the same